### PR TITLE
Remove node label from cnv_abnormal

### PR DESCRIPTION
**Pull request by avlitman**
**What this PR does / why we need it**:
It's not possible to add the node label to telemetry, this pr removes it. 

**Jira Ticket**:

```jira-ticket: 
https://issues.redhat.com/browse/CNV-40387

```

**Release note**:

```release-note
none
```

### CODE REVIEW

File: pkg/monitoring/rules/recordingrules/operator.go [hunk 1]
```diff
@@ -29,6 +29,6 @@ var operatorRecordingRules = []operatorrules.RecordingRule{
 			Help: "Monitors resources for potential problems",
 		},
 		MetricType: operatormetrics.GaugeType,
-		Expr:       intstr.FromString(`sum by (container, node, reason)(kubevirt_memory_delta_from_requested_bytes)`),
+		Expr:       intstr.FromString(`sum by (container, reason)(kubevirt_memory_delta_from_requested_bytes)`),
```
Comment (machadovilaca) [inline pkg/monitoring/rules/recordingrules/operator.go:32]: fine by me, but why do we need to remove the node label? cardinality?

Comment (avlitman) [reply pkg/monitoring/rules/recordingrules/operator.go:32]: Yes, cardinality, they say the node names may vary over time which may add too many series to telemeter

Comment (machadovilaca) [reply pkg/monitoring/rules/recordingrules/operator.go:32]: makes sense

**Pull request merged**